### DISCUSSION TITLE
Photoprism

### DISCUSSION THREAD

**Discussion by jccsouza1986** [photoprism/photoprism#4129] [Q&A]
Estou com dificuldade para entrar em contato com a equipe do Photoprism. Eu comprei o Photoprism anteontem e até agora não consegui instalar. Não sei se mandei a pergunta no lugar certo mas espero que alguém me ajude.

**Comment by graciousgrey** [comment]
Hello 👋

I am happy to help you with this. After you have installed PhotoPrism as described here: https://docs.photoprism.app/getting-started/docker-compose/ you can log in with the admin account created during the installation (not with your member account). The username is admin by default and the password is the one you configured as PHOTOPRISM_ADMIN_PASSWORD in the docker-compose.yml file before you started your instance for the first time. This admin account is not connected to your membership account on my.photoprism.app.
To activate your membership, you can log in to your PhotoPrism instance as admin and then follow step 3 of this guide: https://www.photoprism.app/kb/activation
